tests/p2p_tests/sync/test.sh

```diff
@@ -3,20 +3,43 @@
 pnodes=10
 npnodes=0
 topo=star
-if [ -n "$1" ]; then
-    pnodes=$1
-    if [ -n "$2" ]; then
-        topo=$2
-        if [ -n "$3" ]; then
-            npnodes=$3
+delay=0
```

Comment:
Should we test with a delay > 0 ?

Reply:
Sure. I can add a variation to the travis script that does something like: test.sh -p 2 -d 10 
to start two producer instances with a 10 second delay.